src/cogs/help.py

```diff
@@ -40,33 +40,50 @@ def get_bot_help(
         mapping: Mapping[Optional[commands.Cog], List[commands.Command]]
     ) -> discord.Embed:
         """Generates the embed for the bot help command."""
-        out = discord.Embed(title="Help")
+        out = discord.Embed(
+            title="Commands and modules",
+            description="View a specific module or commands info with:\n```ini\n!help [module_name|command_name]```",
```

Comment:
Some strings like the one here are too long, but black in repo is broken right now, I'll probably fix this myself.